nilmtk/dataset_converters/dred/metadata/building1.yaml

```diff
@@ -0,0 +1,81 @@
+instance: 1 
+original_name: house_1   
+elec_meters:
+  1: &dred_whole_house 
+    site_meter: true
+    device_model: mains
+  2: &emonitor
```

Comment:
is the name of their appliance meter emonitor?

Reply:
I didn't find any name mentioned in their official .h5 metadata. What should I change it to?

Reply:
I am changing it to &mains and &GT then. They should be the device model names

Reply:
Make it appliance_monitor if you do not find the name of the appliance monitor they used in the paper.